src/Symfony/Component/Webhook/Tests/Server/TransportTest.php

```diff
@@ -0,0 +1,104 @@
+        $client->expects($this->once())
+            ->method('request')
+            ->with(
+                'POST',
+                'https://example.com/webhook',
+                $this->isType('array')
+            );
+
+        $subscriber = $this->createMock(Subscriber::class);
+        $subscriber->expects($this->once())->method('getSecret')->willReturn('some-secret');
+        $subscriber->expects($this->once())->method('getUrl')->willReturn('https://example.com/webhook');
+
+        $event = $this->createMock(RemoteEvent::class);
+
+        $configuratorOne = $this->createMock(RequestConfiguratorInterface::class);
+        $configuratorOne->expects($this->once())
+            ->method('configure')
+            ->with($event, 'some-secret', $this->isInstanceOf(HttpOptions::class));
+
+        $configuratorTwo = $this->createMock(RequestConfiguratorInterface::class);
+        $configuratorTwo->expects($this->once())
+            ->method('configure')
+            ->with($event, 'some-secret', $this->isInstanceOf(HttpOptions::class));
+
+        $transport = new Transport($client, new \ArrayIterator([
+            $configuratorOne,
+            $configuratorTwo,
+        ]));
+
+        $transport->send($subscriber, $event);
+    }
+
+    /**
+     * @group legacy
+     */
+    public function testCanBeInstantiatedWithDeprecatedOptions()
+    {
+        $client = $this->createMock(HttpClientInterface::class);
+        $client->expects($this->once())
+            ->method('request')
+            ->with(
+                'POST',
+                'https://example.com/webhook',
+                $this->isType('array')
+            );
+
+        $subscriber = $this->createMock(Subscriber::class);
+        $subscriber->expects($this->once())->method('getSecret')->willReturn('some-secret');
+        $subscriber->expects($this->once())->method('getUrl')->willReturn('https://example.com/webhook');
+
+        $event = $this->createMock(RemoteEvent::class);
+
+        $headerConfigurator = $this->createMock(RequestConfiguratorInterface::class);
+        $headerConfigurator->expects($this->once())
+            ->method('configure')
+            ->with($event, 'some-secret', $this->isInstanceOf(HttpOptions::class));
+
+        $bodyConfigurator = $this->createMock(RequestConfiguratorInterface::class);
+        $bodyConfigurator->expects($this->once())
+            ->method('configure')
+            ->with($event, 'some-secret', $this->isInstanceOf(HttpOptions::class));
+
+        $signerConfigurator = $this->createMock(RequestConfiguratorInterface::class);
+        $signerConfigurator->expects($this->once())
+            ->method('configure')
+            ->with($event, 'some-secret', $this->isInstanceOf(HttpOptions::class));
+
+        $event = $this->createMock(RemoteEvent::class);
+
+        $this->expectUserDeprecationMessage('Since symfony/webhook 7.3: Individual configurators for webhook transport is deprecated, use an iterable instead.');
```

Comment:
```suggestion
        $this->expectUserDeprecationMessage('Since symfony/webhook 7.4: Individual configurators for webhook transport is deprecated, use an iterable instead.');
```